notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>These 2 text cells should be combined. Text cell needs a heading.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/825/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>